app/src/main/resources/templates/thymeleaf/views/experiment-page/experiment-description.html

```diff
@@ -0,0 +1,69 @@
+<div id="experimentDescription" class="row column expanded">
+    <div class="media-object stack-for-small" th:classappend="${#strings.startsWith(experimentAccession, 'E-ANND-')} ? 'anndata' : ''">
```

Comment:
Is it possible to move hardcoded experiment accessions like 'E-ANND- ' to a constant file like Backend?

Reply:
Not in this case.
`th:classappend` as its name tells append a string to the `class` property of the `div` element.
In this case, if `experimentAccession` starts with `E-ANND-` then the `class` property's value will be: `media-object stack-for-small anndata` otherwise its value stays `media-object stack-for-small`.
I am not sure what you mean by move it to the backend?
Thanks

Reply:
ok..thanks for the clarification.